setup.py

```diff
@@ -23,171 +24,189 @@
-    if curSystem == "Windows":
-        binDir = "win"
-    elif curSystem == "Linux":
-        binDir = "linux"
-    elif curSystem == "Darwin":
-        binDir = "osx"
+    cur_system = platform.system()
+    if cur_system == "Windows":
+        bin_dir = "win"
+    elif cur_system == "Linux":
+        bin_dir = "linux"
+    elif cur_system == "Darwin":
+        bin_dir = "osx"
     else:
         raise KeyError("Do not recognize target OS: %s" % sys.platform)
     platform_name = get_platform()
-    return binDir, platform_name, curSystem
+    return bin_dir, platform_name, cur_system
 
 
-def compile(pgkDir):
+def compile_package(pkg_dir):
     if 0:
         print("Skipping compile - hard coded.")
         return
-    binDir, platform_name, curSystem = getExecutableDir()
-    programsDir = os.path.join(pgkDir, "Tools", "Programs")
-    if binDir == 'osx':
+    bin_dir, platform_name, cur_system = get_executable_dir()
+    programs_dir = os.path.join(pkg_dir, "Tools", "Programs")
+    cmd = None
+    if bin_dir == 'osx':
         cmd = "sh BuildAll.sh"
-    elif binDir == 'win':
+    elif bin_dir == 'win':
         cmd = "BuildAll.cmd"
-    elif binDir == 'linux':
+    elif bin_dir == 'linux':
         cmd = "sh BuildAllLinux.sh"
     curdir = os.getcwd()
-    subprocess.check_call(cmd, cwd=programsDir, shell=True)
+    assert cmd, 'Unable to form command for this platform.'
+    subprocess.check_call(cmd, cwd=programs_dir, shell=True)
     os.chdir(curdir)
 
 
+# noinspection PyClassicStyleClass
 class CustomBuild(setuptools.command.build_py.build_py):
     """Custom build command."""
     def run(self):
-        pgkDir = 'afdko'
-        compile(pgkDir)
+        pkg_dir = 'afdko'
+        compile_package(pkg_dir)
         setuptools.command.build_py.build_py.run(self)
 
 
-binDir, platform_name, curSystem = getExecutableDir()
-pkg_list = find_packages()
-# Build the install path for the tools.
-python_bin = "%s/%s" % (sys.prefix, 'bin')
-classifiers = [
-    'Development Status :: 5 - Production/Stable',
-    'Intended Audience :: Developers',
-    'Topic :: Software Development :: Build Tools',
-    'License :: OSI Approved :: Apache Software License',
-    'Programming Language :: Python :: 2.7',
-]
-"""
-Identify the dist build as being platform specific.
-"""
-scripts = [
-    'afdko/Tools/%s/autohintexe' % binDir,
-    'afdko/Tools/%s/detype1' % binDir,
-    'afdko/Tools/%s/makeotfexe' % binDir,
-    'afdko/Tools/%s/mergeFonts' % binDir,
-    'afdko/Tools/%s/rotateFont' % binDir,
-    'afdko/Tools/%s/sfntdiff' % binDir,
-    'afdko/Tools/%s/sfntedit' % binDir,
-    'afdko/Tools/%s/spot' % binDir,
-    'afdko/Tools/%s/tx' % binDir,
-    'afdko/Tools/%s/type1' % binDir,
-]
-
-if curSystem == "Darwin":
-    moreKeyWords = ['Operating System :: MacOS :: MacOS X']
-elif curSystem == "Windows":
-    moreKeyWords = ['Operating System :: Microsoft :: Windows']
+def get_console_scripts():
+    console_scripts = [
```

Comment:
Remove the `return` from the bottom and prepend it here.

Reply:
I'm going to clean this up much more than that.